### ISSUE TITLE
Figure out how to do local dev while serving console from Nexus

### ISSUE THREAD

**Issue by david-crespo** [oxidecomputer/console#582]
We like to use the Vite dev server for local development because of hot module reloading. However, there are some features implemented in Nexus that we're starting to rely on. For example, in https://github.com/oxidecomputer/omicron/pull/541 I'm adding a `GET /login` endpoint to Nexus that will redirect to whatever the actual login form URL is (an IdP page in production, `/spoof_login` in dev for now). But that doesn't work if we're serving the console from the dev server instead of Nexus because we never hit Nexus for `GET /login` — the dev server just returns the JS app. In #581 I simulate that with a client-side redirect from `/login` to `/spoof_login`, but that is not a general solution and I think we're going to run into this in places that are not as easy to hack around.

Basically what I want is to be able to serve the console from Nexus but also get hot reloading. That may not be possible, so instead I would take automatic bundle rebuild requiring a manual page refresh.
